### DISCUSSION TITLE
Preparing the v7.1.1 release

### DISCUSSION THREAD

**Discussion by vmiklos** [vmiklos/plees-tracker#119] [General]
Hi,

I plan to release v7.1.1 towards the end of the next week. It currently contains some English-only strings.

@sguinetti @fabianski7 may I ask you to update the translations you already created in the past?

Test APK here: https://github.com/vmiklos/plees-tracker/releases/tag/v7.1.1-rc1

It contains some nice improvements by @SebiderSushi and @usashiki

Thanks,

Miklos

**Comment by fabianski7** [comment]
well, I couldn't install this version of the app. I made a backup of the previous one and uninstalled it but still android does not allow.
maybe a signed version is needed? I'm using android 11

**Reply by vmiklos** [reply]
Try https://share.vmiklos.hu/plees-tracker/app-debug-v7.1.1-rc1.apk instead, it's a self-signed one.

**Comment by sguinetti** [comment]
Nice to know. I create this PR #120.

**Comment by fabianski7** [comment]
Done. Sorry for the delay
